changes/6247.dependencies

```diff
@@ -0,0 +1 @@
+Updated `mkdocs-material` to `~9.5.35`
```

Comment:
I think this belongs in `.housekeeping` since it's not a prod dependency?

```suggestion
Updated documentation dependency `mkdocs-material` to `~9.5.35`.
```